bin/ci/ubuntu/install-dependencies.sh

```diff
@@ -13,16 +18,34 @@ done
 if [[ -n ${CLANG_VER:-} ]]; then
     # There are cases where the directory exists, but the exe is not available.
     # Use this workaround for now.
-    if [[ ! -x llvm-${LLVM_VERSION}/bin/llvm-config ]] && [[ -d llvm-${LLVM_VERSION} ]]; then
-        rm -fr llvm-${LLVM_VERSION}
+    if [[ ! -x ${TWD}/llvm-${LLVM_VERSION}/bin/llvm-config ]] && [[ -d ${TWD}/llvm-${LLVM_VERSION} ]]; then
```

Comment:
FYI: `[[` can handle `&&`. So the above could be re-written with a single  `[[ ... ]]`. (fine as is though)